fastlane/lib/fastlane/actions/modify_services.rb

```diff
@@ -158,9 +158,12 @@ def self.example_code
             app_identifier: "com.someorg.app",
             services: {
               push_notification: "on",
-              associated_domains: "off"
-            }
-          )'
+              associated_domains: "off",
+              wallet: :on,
+              apple_pay: :off,
+              data_protection: true,
+              multipath: false
```

Comment:
How come we are adding some of these in here? 🙃

Reply:
I put an example of each value type, the way you can enable or disable services